.github/stale.yml

```diff
@@ -0,0 +1,37 @@
+# Default values: https://probot.github.io/apps/stale/#installation
+
+daysUntilStale: 180
+
+exemptLabels:
+  - C-tracking-issue      # keep tracking issues open
+  - "Feature accepted"    # keep accepted features
+
+staleLabel: stale
+
+markComment: >
+  As there hasn't been any recent activity here for 180 days this has been
+  marked as stale, and if no further activity happens for 7 days this will be
```

Comment:
I see that there has not been any activity in over 180 days, can I close this issue?

Reply:
If the"I" is being overused then maybe:
```
Move to close? As there has not been activity in over 180 days. If there is no response in 7 days this will be closed for now.

The teem would appreciate if anyone could give an update on:
```